adapter-commons/src/main/resources/schema/commons.xml

```diff
@@ -5,7 +5,7 @@
     <changeSet author="jakubpilimon" id="1.init">
        <createSequence sequenceName="hibernate_sequence" startValue="1" incrementBy="1" cacheSize="1"/>
     </changeSet>
-    <changeSet author="jakubpilimon" id="2.postgres.json" dbms="postgresql">
-        <sql>CREATE CAST (character varying AS jsonb) WITH INOUT AS ASSIGNMENT</sql>
-    </changeSet>
+    <!--<changeSet author="jakubpilimon" id="2.postgres.json" dbms="postgresql">-->
```

Comment:
commenting changes that might have already been run somewhere will cause problems :) I know you had problems with jsonb but you can at least add ignore on error for this changelog

Reply:
Feel free to fix it